contracts/UniswapV2Factory.sol

```diff
@@ -46,4 +47,15 @@ contract UniswapV2Factory is IUniswapV2Factory {
         require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
         feeToSetter = _feeToSetter;
     }
+    
+    function setProtocolFee(uint8 _protocolFeeDenominator) external {
+        require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
+        require(_protocolFeeDenominator > 0, 'UniswapV2: FORBIDDEN_FEE');
+        protocolFeeDenominator = _protocolFeeDenominator;
+    }
+    
+    function setSwapFee(address pair, uint8 swapFee) external {
```

Comment:
- swapFee -> swapFeeDenominator
- pair > _pair, swapFee > _swapFeeDenominator

Reply:
Makes sense, to follow the same parameter naming we use in the setters.